src/content/developers/docs/consensus-mechanisms/pow/index.md

```diff
@@ -3,76 +3,73 @@ title: Proof-of-work (PoW)
 
 ## Proof-of-work and mining {#pow-and-mining}
 
-Proof-of-work is the underlying algorithm that sets the difficulty and rules for the work miners do. Mining is the "work" itself. It's the act of adding valid blocks to the chain. This is important because the chain's length helps the network follow the correct Ethereum chain and understand Ethereum's current state. The more "work" done, the longer the chain, and the higher the block number, the more certain the network can be of the current state of things.
+Proof-of-work is the underlying algorithm that sets the difficulty and rules for the work miners do on proof-of-work blockchains. Mining is the "work" itself. It's the act of adding valid blocks to the chain. This is important because the chain's length helps the network follow the correct fork of the blockchain. The more "work" done, the longer the chain, and the higher the block number, the more certain the network can be of the current state of things.
 
 [More on mining](/developers/docs/consensus-mechanisms/pow/mining/)
 
-## How does Ethereum's proof-of-work work? {#how-it-works}
+## How did Ethereum's proof-of-work work? {#how-it-works}
 
-Ethereum transactions are processed into blocks. Each block has a:
+Ethereum transactions are processed into blocks. In proof-of-work Ethereum, each block contained:
 
 - block difficulty – for example: 3,324,092,183,262,715
 - mixHash – for example: `0x44bca881b07a6a09f83b130798072441705d9a665c5ac8bdf2f39a3cdf3bee29`
 - nonce – for example: `0xd3ee432b4fb3d26b`
 
-This block data is directly related to proof-of-work.
+This block data was directly related to proof-of-work.
 
 ### The work in proof-of-work {#the-work}
 
-The proof-of-work protocol, Ethash, requires miners to go through an intense race of trial and error to find the nonce for a block. Only blocks with a valid nonce can be added to the chain.
+The proof-of-work protocol, Ethash, required miners to go through an intense race of trial and error to find the nonce for a block. Only blocks with a valid nonce could be added to the chain.
 
-When racing to create a block, a miner will repeatedly put a dataset, that you can only get from downloading and running the full chain (as a miner does), through a mathematical function. The dataset gets used to generate a mixHash below a target nonce, as dictated by the block difficulty. The best way to do this is through trial and error.
+When racing to create a block, a miner repeatedly put a dataset, that could only be obtained by downloading and running the full chain (as a miner does), through a mathematical function. The dataset was used to generate a mixHash below a target nonce, as dictated by the block difficulty. The best way to do this is through trial and error.
```

Comment:
> The dataset was used to generate a mixHash **below a target nonce**

This is a little confusing... the _nonce itself_ can be as high as it needs to be to allow the resulting block _hash_ to be less than a calculated target.

Reply:
yep, thanks for the catch.
```suggestion
When racing to create a block, a miner repeatedly put a dataset, that could only be obtained by downloading and running the full chain (as a miner does), through a mathematical function. The dataset was used to generate a mixHash below a target that is dictated by the block difficulty. The best way to do this is through trial and error.
```